pycrunch/lemonpy.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, session):
     def __call__(self, r, *args, **kwargs):
         code = r.status_code
 
+        if code in (301, 302):
```

Comment:
The recommended way to do this is `def status_301(self, r): return r` like we do for other codes below. This allows consumer code to override more easily if needed.